tests/unittests/HyphenTest.cpp

```diff
@@ -32,6 +32,7 @@
 #include "gtest/gtest.h"
 
 #include <string>
+#include <cctype>

```

Comment:
assuming it's needed for `std::islower`. Please update the description of the PR on what this "include" fixes.